language tutorials/py.html

```diff
@@ -59,7 +59,6 @@ <h1>What can Python Do?</h1>
 			<li>Python can be used alongside software to create workflows.</li>
 			<li>Python can connect to database systems. It can also read and modify files.</li>
 			<li>Python can be used to handle big data and perform complex mathematics.</li>
-			<li>Python can be used to handle big data and perform complex mathematics.</li>
```

Comment:
I was good to know that you made through the project and deleted this bug,, thanks to moiz